setup.py

```diff
@@ -55,9 +55,11 @@
         'jupyter',
         'ipykernel',
         'flatten_json',
+        'cryptography==3.4.6',
         'cryptography>=3.4.6',
-        'six'  # Remove this after upgrade to torchvision==0.9
+        'six',  # Remove this after upgrade to torchvision==0.9
         # See https://github.com/pytorch/vision/issues/3497
+        'pytest'
```

Comment:
Could we re-use [requirements-test.txt](https://github.com/intel/openfl/blob/develop/requirements-test.txt), instead of adding _pytest_ dependency to setup.py ?

Reply:
Sure thing! I was putting stuff into `setup` to reduce the amount of code on the Azure pipelines code. 😄